changelog/README.md

```diff
@@ -8,6 +8,10 @@ For more details, see [Environments](../guidelines/environments.md).
 
 # Changelog
 
+## 7th September 2022
+
+* Described the [Pagination](../guidelines/pagination.md) for the [Customers Get all](../operations/Customers.md) endpoint.
```

Comment:
```suggestion
* Described the [Pagination](../guidelines/pagination.md) for the [Customers Get all](../operations/customers.md#get-all-customers) endpoint.
```

Reply:
I would separate it into two changes-

1. That you have updated pagination description
2. That you have implemented that new pagination into the endpoint

@MikeAdamsMews what do you think?